dbms/src/Interpreters/Join.h

```diff
@@ -34,6 +34,8 @@
 #include <memory>
 #include <shared_mutex>
 
+#include "Interpreters/Users.h"
```

Comment:
Format the code and why need to include this header file?

Reply:
Sorry, it is automatically written by vscode and has been deleted.